blocks/library/shortcode/index.js

```diff
@@ -72,14 +68,11 @@ export const settings = {
 						<Dashicon icon="editor-code" />
 						{ __( 'Shortcode' ) }
 					</label>
-					<TextareaAutosize
+					<PlainText
 						id={ inputId }
-						autoComplete="off"
```

Comment:
Was it intended change to enable `autoComplete`?

Reply:
Yes, this is not needed anymore as it's not an input